paper-server/src/main/java/org/bukkit/craftbukkit/inventory/view/builder/CraftAbstractInventoryViewBuilder.java

```diff
@@ -0,0 +1,56 @@
+package org.bukkit.craftbukkit.inventory.view.builder;
+
+import com.google.common.base.Preconditions;
+import io.papermc.paper.adventure.PaperAdventure;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.MenuType;
+import org.bukkit.craftbukkit.entity.CraftHumanEntity;
+import org.bukkit.craftbukkit.util.CraftChatMessage;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.inventory.InventoryView;
+import org.bukkit.inventory.view.builder.InventoryViewBuilder;
+import org.jetbrains.annotations.NotNull;
+
+public abstract class CraftAbstractInventoryViewBuilder<V extends InventoryView> implements InventoryViewBuilder<V> {
+
+    protected final MenuType<?> handle;
+
+    protected boolean checkReachable = false;
+    protected Component title = null;
+
+    public CraftAbstractInventoryViewBuilder(MenuType<?> handle) {
+        this.handle = handle;
+    }
+
+    @NotNull
+    @Override
+    public InventoryViewBuilder<V> title(@NotNull final String title) {
+        this.title = LegacyComponentSerializer.legacySection().deserialize(title);
+        return this;
+    }
+
+    @Override
+    public @NotNull InventoryViewBuilder<V> title(final @NotNull Component title) {
```

Comment:
title should not be forced
vanilla has for every inventory type a title already which should be displayed if title is null